dashboard/src/features/unspents/reducers.js

```diff
@@ -2,7 +2,7 @@ import { reducers } from 'features/shared'
 import { combineReducers } from 'redux'
 
 const type = 'unspent'
-const idFunc = item => `${item.output_id}`
+const idFunc = item => `${item.id}`
```

Comment:
You can just delete `idFunc` and remove it as a parameter on line 8. `id` is the default.